Files/ViewModels/SettingsViewModel.cs

```diff
@@ -847,7 +841,7 @@ public TValue Get<TValue>(TValue defaultValue, [CallerMemberName] string propert
 
         public void Dispose()
         {
-            DrivesManager?.Dispose();
+            App.DrivesManager?.Dispose();
```

Comment:
Do we still need this reference?

Reply:
Agreed, this Dispose method can go away. In `OnSuspending` change `AppSettings?.Dispose();` to `DrivesManager?.Dispose()`

Reply:
Updated